aquarius/events/metadata_updater.py

```diff
@@ -268,13 +277,16 @@ def get_datatoken_pools(self, dt_address, from_block=0, to_block='latest'):
             return None
 
         pools = [get_event_data(event_abi, log).address for log in logs]
-        return pools
+        return list(set(pools))
```

Comment:
remove `set` after making the change above.

Reply:
see above